### DISCUSSION TITLE
Proposal for New Features on Quran.com

### DISCUSSION THREAD

**Discussion by SyedaMahamFahim** [quran/discussions#14] [Ideas]
**Assalam o Alaikum Quran.com Team,**

I hope this message finds you well. I am an active user of Quran.com, and my current streak is 30 days. The platform has significantly helped me in maintaining a daily Quran reading habit, in addition to my regular recitations during Fajr and Isha.

I am writing to propose two features that I believe would enhance the user experience, particularly for female users, and to offer my assistance in implementing these features.

### **1. Period Days for Women**

As a female user, I anticipate having menstrual periods, which may affect my ability to maintain my streak. Currently, if I am unable to read the Quran due to my period, my streak will be broken. To address this, I propose the following feature:

- **Feature Description**: Allow female users to input their period start date and duration (up to 7 days). During this period, their streak should be preserved. However, if the user does not complete their reading on the 8th day or later, their streak will be broken.
- **Benefit**: This feature would accommodate the unique needs of female users, allowing them to maintain their spiritual routine without the stress of streak interruption due to their period.

### **2. Grouping Surahs for Easy Access**

I regularly recite specific Surahs during Fajr and Isha and would find it helpful to have a feature to group these Surahs for quick access:

- **Feature Description**: Implement a grouping feature where users can create custom groups of Surahs. Users can name these groups and add Surahs to them. The feature would also display the dates on which these Surahs were recited, including Fridays.
- **Benefit**: This would streamline the process of accessing frequently recited Surahs and enhance the overall user experience by reducing the time spent searching for specific Surahs.

### **Contribution Offer**

As a Software Engineer with experience in Next.js, I am eager to contribute to the development of these features. I believe this could be a valuable addition to Quran.com and a way for me to contribute to a project that I deeply value. If the Quran.com team is open to this proposal, I would be delighted to assist in developing and implementing these features.

Thank you for considering my proposal. I look forward to your feedback and am happy to discuss any details further.

**Best regards,**

Syeda Maham Fahim  
syedamahamfahim@gmail.com

**Comment by muhdhasnuddin-max** [comment]
I wish the moderators can reply to this request. It seems like its been a year and no respond was written in reply to this sister request.
